### PR TITLE
Small changes to README

### PR DESCRIPTION
**Pull request by franbh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,9 +19,9 @@ Hopefully, it will get better soon.
     * It should look like:
     ```
     [github]
-       token = "my_token"
+       token = my_token
```
Comment (leobenkel) [inline README.md:22]: it works with the quotes. that's what i have in my file

File: README.md [hunk 2]
```diff
@@ -19,9 +19,9 @@ Hopefully, it will get better soon.
...
     ```
-3. Store token under `GITHUB_TOKEN` in the environment.
+   Alternatively, you can store the token under `GITHUB_TOKEN` in the environment.
```
Comment (leobenkel) [inline README.md:24]: I am not sure if the env variable is not needed. it was not always working only with the config

**Review by leobenkel — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (franbh) [reply README.md:22]: I think it doesn't work, that's why you need the `GITHUB_TOKEN` env var

File: README.md (hunk 2 above)
Comment (franbh) [reply README.md:24]: see above comment

**Review by leobenkel — APPROVED** [review]

**Pull request merged**